From 5e2f8dab9e8777135980b9ff896752803f2b25db Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 1 Sep 2022 11:51:53 +0200
Subject: [PATCH] arm64: defconfig: enable NVMEM STM32 ROMEM for stm32mp25

Enable the NVMEM driver STM32 ROMEM for STM32MP25x SoCs
to allow access to OTP provided by OP-TEE BSEC PTA

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I5071bb8c60fb59975321f211f16857fbe631bd36
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361798
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -7,6 +7,7 @@ CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_MFD_STM32_TIMERS=m
+CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_OF_OVERLAY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PWM_STM32=m
